src/pages/Trends/TrendsPage.module.scss

```diff
@@ -46,3 +39,77 @@
   }
 }
 /* stylelint-enable */
+
+.column {
+  display: flex;
+
+  &__left {
+    flex: 2;
+    margin-right: 30px;
+  }
+
+  &__right {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.subtitle {
+  font-family: Rubik;
```

Comment:
@DmitriVanGuard Do we need to have this line?

Reply:
Unfortunately - yes. There are global styles that should be removed in future css refactoring PR (it's a breaking change) 
![image](https://user-images.githubusercontent.com/25135650/55542439-b9fb8b80-56cf-11e9-92e0-e9947972215c.png)